test/unit/vagrant/action/runner_test.rb

```diff
@@ -27,6 +28,10 @@ def action(env)
       def initialize(app, env)
       end
 
+      def self.name
+        "TestAction"
+      end
+
```

Comment:
We probably should include some tests where the action is an instance of an anonymous class that does not have `self.name` properly defined (and returns a `nil` value). Mainly to guard against any places where it may already be in use, or where it may be used in the future (thinking most of plugins).